### ISSUE TITLE
webCrypto error in Node v16 [REPL]

### ISSUE THREAD

**Issue by coolaj86** [perry-mitchell/ulidx#4]
```bash
npm install --save ulidx
```

```js
let ulid = require('ulidx').ulid;
ulid();
```

```txt
Uncaught TypeError: webCrypto.getRandomValues is not a function
    at /home/app/node_modules/ulidx/dist/ulid.js:60:23
    at randomChar (/home/app/node_modules/ulidx/dist/ulid.js:167:27)
    at encodeRandom (/home/app/node_modules/ulidx/dist/ulid.js:87:15)
    at Object.ulid (/home/app/node_modules/ulidx/dist/ulid.js:184:41)
```

**Comment by perry-mitchell** [comment]
Tests are passing for [node 16](https://github.com/perry-mitchell/ulidx/runs/3703983740?check_suite_focus=true) - Would you suggest another test be added to the suite?

**Comment by perry-mitchell** [comment]
That might be it. This library uses the existence of `crypto` on global to detect web vs node. In standard node execution crypto is not available globally, so I'd say that the REPL is misrepresenting its environment.

It's a pity, because imo it's completely incorrect. That being said supporting the REPL is still important. I'll leave this open to track the fix.

**Comment by perry-mitchell** [comment]
Fixed in 0.3.0
